src/Core/Domain/Address/QueryResult/EditableCustomerAddress.php

```diff
@@ -107,7 +109,7 @@ class EditableCustomerAddress
     private $address2;
 
     /**
-     * @var StateId|null
+     * @var StateIdInterface|null
```

Comment:
Same suggestion here, the `$stateId` parameter is always provided so it seems `$this->stateId` can never be null